edgeai-apps-utils/suite/bookworm/debian/libedgeai-apps-utils-dev.postinst

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p /opt
+
+git clone https://git.ti.com/git/edgeai/edgeai-apps-utils /opt/edgeai-apps-utils
```

Comment:
Nak, just copy the source to somewhere more standard, like `/usr/src` or `/usr/share`. Have it as part of the package content, no .postinst downloading/magic please, packages should install the same without network access.